### PR TITLE
Print node status even in --quiet mode

### PR DESCRIPTION
**Pull request by poornas**
## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#97f704ce-24a5-41ee-9958-4b162d86372a) |
<a name="97f704ce-24a5-41ee-9958-4b162d86372a"></a>
### 9149-3db896a/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c4.minio.io:32334
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp ae32c5a7a473:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 6 minutes and 50 seconds
(4/15) Running aws-sdk-ruby tests ... done in 18 seconds
(5/15) Running awscli tests ... FAILED in 3 minutes and 22 seconds
{
  "name": "awscli",
  "duration": 1569,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Review by harshavardhana — CHANGES_REQUESTED** [review]
We should also enhance the output to include if the nodes are `offline` users should be guided to use `mc admin info` to get the latest information.

**Comment by kannappanr** [comment]
Did not work in `fs` mode

**Comment by poornas** [comment]
> We should also enhance the output to include if the nodes are `offline` users should be guided to use `mc admin info` to get the latest information.

done

**Comment by poornas** [comment]
> Did not work in `fs` mode

this is displayed only for erasure backend.

File: cmd/server-startup-msg.go [hunk 1]
```diff
@@ -249,8 +250,12 @@ func printObjectAPIMsg() {
 // Get formatted disk/storage info message.
 func getStorageInfoMsgSafeMode(storageInfo StorageInfo) string {
 	var msg string
+	var mcMessage string
 	if storageInfo.Backend.Type == BackendErasure {
-		diskInfo := fmt.Sprintf(" %d Online, %d Offline. ", storageInfo.Backend.OnlineDisks.Sum(), storageInfo.Backend.OfflineDisks.Sum())
+		if storageInfo.Backend.OfflineDisks.Sum() > 0 {
+			mcMessage = "Run `mc admin info` to check server info`"
```
Comment (harshavardhana) [inline cmd/server-startup-msg.go]:
```suggestion
			mcMessage = "%s offline nodes detected during startup, please use `mc admin info` to look for latest info"
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-fs.sh | :x: [more...](#d742629f-bca9-42de-8949-e88106fba7b0) |
| mint-gateway-azure.sh | :x: [more...](#e4a2a14e-03de-44d6-803a-bad6679373a7) |
<a name="e4a2a14e-03de-44d6-803a-bad6679373a7"></a>
### 9149-f21d504/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:30331
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 89377db6a30f:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 9 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 6 minutes and 27 seconds
(4/15) Running aws-sdk-ruby tests ... done in 18 seconds
(5/15) Running awscli tests ... FAILED in 1 minutes and 37 seconds
{
  "name": "awscli",
  "duration": 1536,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
<a name="d742629f-bca9-42de-8949-e88106fba7b0"></a>
### 9149-f21d504/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:32200
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 071ebc4002d0:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 1 minutes and 7 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... FAILED in 16 seconds
{
  "name": "mc",
  "duration": "10072",
  "function": "test_presigned_post_policy_error",
  "status": "FAIL",
  "error": "unable to get presigned post policy and put object url"
}

Executed 6 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#28642b6b-8170-4fc3-a7d6-a507803ac66e) |
<a name="28642b6b-8170-4fc3-a7d6-a507803ac66e"></a>
### 9149-f3fa9c5/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev4.minio.io:30509
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9154f2f9e665:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 6 minutes and 47 seconds
(4/15) Running aws-sdk-ruby tests ... done in 19 seconds
(5/15) Running awscli tests ... FAILED in 39 seconds
{
  "name": "awscli",
  "duration": 2529,
  "function": "aws --endpoint-url http://minio-dev4.minio.io:30509 s3api put-object --body /mint/data/datafile-1-kB --bucket \nAn error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it. --key datafile-1-kB\n",
  "status": "FAIL",
  "error": "usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]To see help text, you can run:  aws help  aws <command> help  aws <command> <subcommand> helpUnknown options: occurred, (BucketAlreadyOwnedByYou), when, calling, the, CreateBucket, operation:, Your, previous, request, to, create, the, named, bucket, succeeded, and, you, already, own, it., error"
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#810b76d9-f8dc-4f36-952b-14686bf5313d) |
| mint-gateway-azure.sh | :x: [more...](#7aada2cd-6d5c-4177-a0f6-d0f41b3926f0) |
<a name="810b76d9-f8dc-4f36-952b-14686bf5313d"></a>
### 9149-d3b6c9a/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:31441
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 5a832de33133:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 46 seconds
(4/15) Running aws-sdk-ruby tests ... done in 5 seconds
(5/15) Running awscli tests ... done in 2 minutes and 15 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 39 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 2 seconds
(9/15) Running minio-go tests ... FAILED in 13 minutes and 12 seconds
{
  "args": {},
  "duration": 45993,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 15 tests successfully.
```
<a name="7aada2cd-6d5c-4177-a0f6-d0f41b3926f0"></a>
### 9149-d3b6c9a/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:30748
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d3f99d34e949:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 6 minutes and 51 seconds
(4/15) Running aws-sdk-ruby tests ... done in 18 seconds
(5/15) Running awscli tests ... FAILED in 2 minutes and 58 seconds
{
  "name": "awscli",
  "duration": 1485,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally
Error: No such image: minio/minio:9149-d3b6c9a

File: cmd/server-startup-msg.go [hunk 2]
```diff
@@ -259,8 +264,12 @@ func getStorageInfoMsgSafeMode(storageInfo StorageInfo) string {
 // Get formatted disk/storage info message.
 func getStorageInfoMsg(storageInfo StorageInfo) string {
 	var msg string
+	var mcMessage string
 	if storageInfo.Backend.Type == BackendErasure {
-		diskInfo := fmt.Sprintf(" %d Online, %d Offline. ", storageInfo.Backend.OnlineDisks.Sum(), storageInfo.Backend.OfflineDisks.Sum())
+		if storageInfo.Backend.OfflineDisks.Sum() > 0 {
+			mcMessage = "Use `mc admin info` to look for latest info"
```
Comment (harshavardhana) [inline cmd/server-startup-msg.go]: This message can be same as above

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/server-startup-msg.go [hunk 3]
```diff
@@ -249,8 +250,12 @@ func printObjectAPIMsg() {
 // Get formatted disk/storage info message.
 func getStorageInfoMsgSafeMode(storageInfo StorageInfo) string {
 	var msg string
+	var mcMessage string
 	if storageInfo.Backend.Type == BackendErasure {
-		diskInfo := fmt.Sprintf(" %d Online, %d Offline. ", storageInfo.Backend.OnlineDisks.Sum(), storageInfo.Backend.OfflineDisks.Sum())
+		if storageInfo.Backend.OfflineDisks.Sum() > 0 {
+			mcMessage = "Run `mc admin info` to look for latest info`"
```
Comment (harshavardhana) [inline cmd/server-startup-msg.go]: Perhaps we have to say 
```suggestion
			mcMessage = "Run `mc admin info` to look for server/disk info`"
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#e66021fe-acd9-4cb5-80b1-99bf32856771) |
| mint-gateway-azure.sh | :x: [more...](#d10dcdfa-79f3-4e68-9b72-9f64ac732ef6) |
<a name="e66021fe-acd9-4cb5-80b1-99bf32856771"></a>
### 9149-148d9d1/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:31172
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 3d274480c5de:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 47 seconds
(4/15) Running aws-sdk-ruby tests ... done in 5 seconds
(5/15) Running awscli tests ... done in 1 minutes and 24 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 38 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 37 seconds
(9/15) Running minio-go tests ... FAILED in 13 minutes and 46 seconds
{
  "args": {},
  "duration": 51009,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 15 tests successfully.
```
<a name="d10dcdfa-79f3-4e68-9b72-9f64ac732ef6"></a>
### 9149-148d9d1/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:32483
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4dd12aa7bcd0:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 9 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 6 minutes and 54 seconds
(4/15) Running aws-sdk-ruby tests ... done in 18 seconds
(5/15) Running awscli tests ... FAILED in 1 minutes and 52 seconds
{
  "name": "awscli",
  "duration": 1485,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally
Error: No such image: minio/minio:9149-148d9d1

**Review by kannappanr — APPROVED** [review]

**Pull request merged**